sopel/modules/search.py

```diff
@@ -73,38 +74,43 @@ def duck_api(query):
         return None
 
 
-@commands('duck', 'ddg', 'g')
+@plugin.command('duck', 'ddg', 'g')
 # test for bad Unicode handling in py2
-@example(
+@plugin.example(
     '.duck site:grandorder.wiki chulainn alter',
     r'https?:\/\/grandorder\.wiki\/C%C3%BA_Chulainn.*',
     re=True,
     online=True,
     vcr=True)
 # the last example (in source line order) is what .help displays
-@example(
+@plugin.example(
     '.duck sopel.chat irc bot',
     r'https?:\/\/.*sopel.*',
     re=True,
     online=True,
     vcr=True)
+@plugin.output_prefix(PLUGIN_OUTPUT_PREFIX)
 def duck(bot, trigger):
     """Queries DuckDuckGo for the specified input."""
     query = trigger.group(2)
     if not query:
-        return bot.reply('.ddg what?')
+        bot.reply('{prefix}{command} what?'.format(
+            prefix=bot.settings.core.help_prefix,
+            command=trigger.group(1),
```

Comment:
We have unit tests to prove that `trigger.group(1)` is exactly what we need here. It bothered me that we had the ".command what?" everywhere else but here. So it's fixed.